RecoJets/JetProducers/plugins/MVAJetPuIdProducer.h

```diff
@@ -0,0 +1,50 @@
+#include "FWCore/Framework/interface/stream/EDProducer.h"
```

Comment:
please move the contents of this file to MVAJetPuIdProducer.cc
.h is needed if the code is *ever* needed to be included in more than one place. This one doesn't.